specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-07-01/examples/Compute/createOrUpdate/AKSCompute.json

```diff
@@ -0,0 +1,53 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "workspaceName": "workspaces123",
+    "computeName": "compute123",
+    "api-version": "2021-07-01",
+    "parameters": {
+      "location": "eastus",
+      "properties": {
+        "description": "some compute",
+        "resourceId": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourcegroups/testrg123/providers/Microsoft.ContainerService/managedClusters/compute123-56826-c9b00420020b2",
+        "computeType": "AKS",
+        "properties": {
+          "agentCount": 4
```

Comment:
>agentCount [](http://example.com/codeflow?start=11&length=10)

Why is `agentCount` missing from the responses?

Reply:
And why do you have `properties` nested twice?

Reply:
Hi, Marcin, we didn't update this example, it's inherited from previous versions, I'm not sure why it's defined like that, do we have to update this?

Reply:
Does the API behave like the example suggests or is there a bug in the example?

Reply:
agent count should be in responses. and yes, properties are nested twice. It's how the api has been defined since beginning.

Reply:
fixed the example